docs/agent/config.md

```diff
@@ -86,6 +86,7 @@ in any way.
 | `syslog_host`  | `syslog_uri`  | The Syslog configuration is now expressed as an URI |
 || `syslog_pem`  | Syslog configuration client certificate for TLS client validation |
 || `syslog_key`  | Syslog configuration client private key for TLS client validation |
+| `DD_TAGS` | `DD_TAGS` | The format is space-separated, i.e. `simple-tag-0 tag-key-1:tag-value-1` |
```

Comment:
we should explicitly mention that DD_TAGS is an env var, because the corresponding option in the `datadog.yaml` file is `tags`.

Something like this could work:

```suggestion
| `DD_TAGS` | `DD_TAGS` | Environment variable. The format is now space-separated (e.g. `simple-tag-0 tag-key-1:tag-value-1`) instead of comma-separated (e.g. `simple-tag-0,tag-key-1:tag-value-1`) |
```

Reply:
added the env var note, and then rewrote your suggestion so that it all occurs in present-tense